### PR TITLE
1900 show rfi attachments

### PR DESCRIPTION
**Pull request by marcellmueller**
- feat: add sort rfi file util function
- feat: display rfi files on application page

Implements #1900

### CODE REVIEW

File: app/components/Review/widgets/FileWidget.tsx [hunk 1]
```diff
@@ -1,22 +1,144 @@
...
+        if (!isFileField) return null;
+
+        const attachments = rfi?.attachments?.nodes;
+
+        return (
+          <StyledFile key={rfi}>
+            {rfiFiles[fieldName]?.map((el, index) => {
+              // loop through the list of files for the current field
+              const isSingleFile = !options?.allowMultipleFiles;
+              const isDisplayIcon = isSingleFile && i === 0;
```
Comment (rafasdc) [inline app/components/Review/widgets/FileWidget.tsx:73]: What's the purpose of the i === 0? Wouldn't this prevent the latest RFI to have the star? Something like this? 
image

**Review by rafasdc — COMMENTED** [review]
This is working great, just a small question. I also added a general comment to the issue as no stars will show once this is merged as all rfi files are now multi file uploads.

File: app/components/Review/widgets/FileWidget.tsx (hunk 1 above)
Comment (marcellmueller) [reply app/components/Review/widgets/FileWidget.tsx:73]: In the graphql query we are sorting them by `orderBy: RFI_DATA_ID_DESC` and the `i === 0` is so we display the icon for the first rfi (and newest rfi) that we map. 

The files below the rfis are mapped separately so I am confused as to what you mean. I just double checked and it's working for me:

Screenshot 2023-08-11 at 9 27 56 AM

Comment (rafasdc) [reply app/components/Review/widgets/FileWidget.tsx:73]: Sorry! This makes sense now, I was looking at it from the wrong way.

**Review by rafasdc — APPROVED** [review]

File: app/components/Review/widgets/FileWidget.tsx (hunk 1 above)
Comment (marcellmueller) [reply app/components/Review/widgets/FileWidget.tsx:73]: No worries thanks 🙂

**Pull request merged**